alpine-consul/root/usr/bin/consul-nodes-handler.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# count child directories
+# see: http://stackoverflow.com/a/6781464
+# see: https://github.com/smebberson/docker-alpine/pull/13#issuecomment-161005924
+
+# 0) count child directories
+num_child=`ls -l /etc/consul-template/templates/ | grep -c ^d`
+
+# 1) check if templates exist
+if [ $num_child -gt 0 ]; then
+  # 2) remove `down` to enable service
+  if [ -e "/etc/services.d/consul-template/down" ]; then
```

Comment:
Should this be referencing `/var/run/s6/services/consul-template/down`? I haven't run the code yet, but it seems point in which this code will run is after s6-overlay stage 2. During stage 2 all of the `/etc/services.d/` directories are copied to `/var/run/s6/services/`.